module.exports = {
    authenticate : async( credentials )=>{
        try {
            return null
        } catch (error) {
            console.log("Hubo un error", error)
        }
    }
}